src/lib/statistics/statistics_objects/min_max_filter.cpp

```diff
@@ -158,6 +158,49 @@ bool MinMaxFilter<T>::does_not_contain(const PredicateCondition predicate_condit
       const auto value2 = boost::get<T>(*variant_value2);
       return value >= max || value2 <= min;
     }
+    case PredicateCondition::NotLike:
+    case PredicateCondition::Like: {
+      if constexpr (std::is_same_v<T, pmr_string>) {
+        const auto pattern = boost::get<pmr_string>(variant_value);
+
+        const auto single_char_wildcard_pos = pattern.find_first_of('_');
+        const auto multi_char_wildcard_pos = pattern.find_first_of('%');
+
+        const auto single_valid_char_wildcard_exist = (single_char_wildcard_pos != std::string::npos);
+        const auto multi_char_wildcard_pos_exsists = (multi_char_wildcard_pos != std::string::npos);
+
+        auto wildcard_pos = size_t{};
+
+        if (!single_valid_char_wildcard_exist && !multi_char_wildcard_pos_exsists) {
```

Comment:
In case we have no wildcards, shouldn't we defer to the `Equals`/`NotEquals` implementation?